src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -0,0 +1,243 @@
+    this.planOfArrangement = planOfArrangement
+  }
+  /** Called when prop changes (ie, dialog is shown/hidden). */
+  @Watch('dialog')
+  private async onDialogChanged (val: boolean): Promise<void> {
+    // when dialog is shown, reset notation and validation
+    if (val) {
+      this.notation = ''
+      await Vue.nextTick()
+      this.$refs.notationFormRef.resetValidation()
+
+      // This will make CourtOrderPoa to re-render as it will have a different key every time it opens the dialog.
+      // Source: https://michaelnthiessen.com/force-re-render/
+      this.courtOrderKey++
+    }
+  }
+
+  /**
+   * Emits event to close this dialog.
+   * @param needReload Whether the dashboard needs to be reloaded.
+   */
+  @Emit('close')
+  private emitClose (needReload: boolean): void {
+    this.enableValidation = false
+  }
+
+  /** WIP */
+  /** Saves the current notation. */
+  private async save (): Promise<void> {
+    // prevent double saving
+    if (this.saving) return
+    this.saving = true
+
+    this.enableValidation = true
+    await this.$nextTick()
+    const isNotationFormRefValid = this.$refs.notationFormRef.validate()
+    const isCourtOrderPoaFormRefValid = this.$refs.courtOrderPoaRef.validate()
+    if (!isNotationFormRefValid || !isCourtOrderPoaFormRefValid) {
+      this.saving = false
+      return
+    }
+
+    const data = {
+      filing: {
+        header: {
+          name: this.filingType,
+          date: this.currentDate
+          // certifiedBy: full name,
+          // email: no_one@never.get,
+          // filingId: 1
+        },
+        business: {
+          // foundingDate: 2018-01-01T00:00:00+00:00,
+          identifier: this.entityIncNo
+          // legalName: legal name - Test,
+          // legalType: BC
+        },
+        registrarsNotation: {
+          fileNumber: this.courtOrderNumber,
+          effectOfOrder: (this.planOfArrangement ? 'planOfArrangement' : ''),
+          orderDetails: this.notation
+        }
+      }
+    }
+
+    const url = `businesses/${this.entityIncNo}/filings/`
+    let success = false
+    await axios.post(url, data).then(res => {
+      success = true
+    }).catch(error => {
+      // eslint-disable-next-line no-console
+      console.log('save() error =', error)
+      alert('Could not save your notation. Please try again or cancel.')
+      this.saving = false
+    })
+
+    this.saving = false
+    if (success) this.emitClose(true)
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+::v-deep {
+  #court-order div.pl-2 {
+    padding-left: 0 !important;
+  }
+  #court-order {
+    padding-right: 0 !important;
+    padding-top: 0 !important;
+    margin-top: 0 !important;
+    padding-bottom: 0 !important;
+  }
+  #court-num-form div.v-input__slot {
+    margin-bottom: 0 !important;
+  }
+  #court-order .v-input--checkbox .v-input__slot {
+    margin-bottom: 0 !important;
+  }
+}
+.save-btn {
+  font-weight: 700;
```

Comment:
Let's use `font-weight: bold` instead. This app has only 2 font weights: 400 and 700. Let's just use "normal" and "bold" instead of 400/500/600/700/etc.

Reply:
(I know font-weight is used inconsistently in this app... Some day we should update that.

Reply:
👍 It was changed to bold